hack/windows/Clean-AntreaNetwork.ps1

```diff
@@ -4,9 +4,12 @@
 
   .PARAMETER OVSInstallDir
   OVS installation directory. It is the path argument when using Install-OVS.ps1. The default path is "C:\openvswitch".
+  .PARAMETER RenewIPConfig
+  Renew the ipconfig on the host. The default value is $false.
 #>
 Param(
-    [parameter(Mandatory = $false)] [string] $OVSInstallDir = "C:\openvswitch"
+    [parameter(Mandatory = $false)] [string] $OVSInstallDir = "C:\openvswitch",
+    [parameter(Mandatory = $false)] [bool] $RenewIPConfig   = $false
```

Comment:
Previously it ran by default. Is it expected to change the default behavior?

Reply:
Yes, it is because it might cost several minutes (2min from the issue reporter's observations), and the user uses it in the script before shutting down the Windows host. This time cost is not needed in his case because Windows host will re-configure the IP address after start. Besides, "ipconfig renew" works if the Windows host doesn't configured the IP address automically after removing the HNS Network and the OVS bridge, so we could make it work as an add-on.